client/lib/plans/features-list.tsx

```diff
@@ -518,34 +519,52 @@ export const FEATURES_LIST: FeatureList = {
 
 	[ FEATURE_PREMIUM_THEMES ]: {
 		getSlug: () => FEATURE_PREMIUM_THEMES,
-		getTitle: () => {
+		getTitle: () => i18n.translate( 'Premium themes' ),
+		getIcon: () => <img src={ Theme2Image } alt={ i18n.translate( 'Premium themes' ) } />,
+		getDescription: () => i18n.translate( 'Switch between a collection of premium design themes.' ),
+		getConditionalTitle: ( planSlug ) => {
+			if ( ! planSlug ) {
+				return '';
+			}
 			const localeSlug = i18n.getLocaleSlug();
-			const shouldShowNewString =
-				( localeSlug && config< string >( 'english_locales' ).includes( localeSlug ) ) ||
-				i18n.hasTranslation( 'Unlimited premium themes' );
-
-			return shouldShowNewString
-				? i18n.translate( 'Unlimited premium themes' )
-				: i18n.translate( 'Premium themes' );
+			const shouldShowInPersonalPlan =
```

Comment:
Personal plan isn't a thing anymore, we should either find a generic name or call this `shouldShowInStarterPlan` instead.

Reply:
Yeah, I thought about that too, but we have a `isPersonalPlan()` just a few lines ago, so I considered it'd be less confusing this way than having different terms:

https://github.com/Automattic/wp-calypso/blob/e9693bf796d95e289c4cab96f0545381e86949b1/client/lib/plans/features-list.tsx#L534

Reply:
Maybe we could rename both in a follow-up, just so that the code matches the actual naming of things.

Reply:
Renaming the `isPersonalPlan` function will require a bit more of work, since it's a util from `@automattic/calypso-products` that it's used in tons of places. It should be probably implemented as a clean-up task of the plans rename project.